pkgs/development/interpreters/perl/default.nix

```diff
@@ -194,13 +195,15 @@ in rec {
   # the latest Maint version
   perl528 = common {
     self = perl528;
+    buildPerl = buildPackages.perl528;
```

Comment:
Maybe
```diff
- self = perl528;
+ self = pkgs.perl528;
```
to cover the case when ```pkgs.perl528``` (or ```pkgs.perldevel```) is redefined in the overlay.
Currently, it may result in a buggy behavior, because overriding ```pkgs.perl528``` in overlay affects ```buildPackages.perl528``` but not ```perl528``` which is a local here.